experimental/docker-stacks-and-bundles.md

```diff
@@ -52,7 +52,7 @@ Options:
 Let's deploy the stack created before:
 
 ```bash
-# docker deploy vossibility-stack
+# docker deploy --bundle-file vossibilitystack.dab vossibility
```

Comment:
this doesn't look right still; the file mentioned in earlier steps, and in the output below is called `vossibility-stack.dab` (with a dash)